src/widget/woverview.h

```diff
@@ -170,6 +171,9 @@ class WOverview : public WWidget, public TrackDropTarget {
     QColor m_labelTextColor;
     QColor m_labelBackgroundColor;
     QColor m_endOfTrackColor;
+    QColor m_passthroughOverlayColor;
+    QLabel* m_pPassthroughLabel;
+    QVBoxLayout* m_pPassThroughLayout;
```

Comment:
`setLayout` transfers ownership to the widget, you don't need to save a reference to the layout.

Reply:
so `QVBoxLayout passThroughLayout = new ...QVBoxLayout()` would suffice?

Reply:
```c++
QVBoxLayout* pPassThroughLayout = new QVBoxLayout();
```

Reply:
a got it..